features/support/formatter.rb

```diff
@@ -1,13 +1,14 @@
 # Adapted from https://github.com/tpope/fivemat
 require "cucumber/formatter/progress"
+require_relative "./elapsed_time"
 
 module Ao3Cucumber
```

Comment:
Our Cucumber version was... quite old. Several internals have changed, so I had to update this to get the same information in a slightly different way